src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -398,6 +399,7 @@ namespace wallet_rpc
         KV_SERIALIZE(subaddr_indices)
         KV_SERIALIZE(priority)
         KV_SERIALIZE(mixin)
+        KV_SERIALIZE(ring_size)
```

Comment:
KV_SERIALIZE_OPT, or ring_size will not be initialized, and you'll get crazy high values in some cases.